testsuite/pytests/test_multimeter.py

```diff
@@ -68,44 +152,35 @@ def test_recordables_are_recorded(set_resolution, model):
         assert len(result[r]) == num_data_expected
 
 
-@pytest.mark.parametrize("model", models)
-def test_correct_amount_data_collected(model):
+@pytest.mark.parametrize("model", subset_models_with_rec)
+def test_identical_recording_from_multiple_multimeters(model):
```

Comment:
Comparing to the SLI test, I don't think it it necessary to run this test for all neuron models. Doing it for a single model should suffice and save quite a bit of energy over time. We then also do not need the skip list.